packages/editor/src/client/routes/articleList.tsx

```diff
@@ -143,13 +148,11 @@ export function ArticleList() {
             <Cell dataKey="published">
               {(articleRef: ArticleRefFragment) =>
                 articleRef.published?.publishedAt
-                  ? `${new Date(articleRef.published.publishedAt).toDateString()} ${new Date(
-                      articleRef.published.publishedAt
-                    ).toLocaleTimeString()}`
+                  ? `${dateTimeWithoutSecondsLocalString(
+                      new Date(articleRef.published.publishedAt)
+                    )}`
                   : articleRef.pending?.publishAt
-                  ? `${new Date(articleRef.pending.publishAt).toDateString()} ${new Date(
-                      articleRef.pending.publishAt
-                    ).toLocaleTimeString()}`
+                  ? `${dateTimeWithoutSecondsLocalString(new Date(articleRef.pending?.publishAt))}`
```

Comment:
I recommend to also use the i18next translation service to format the date. Might be a bit more work if it's just the date but it's cleaner this way.